pkgs/tools/misc/turbo/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, git
+, nodejs
+, protobuf
+, protoc-gen-go
+, protoc-gen-go-grpc
+, rustPlatform
+, pkg-config
+, openssl
+, extra-cmake-modules
+, fontconfig
+, go
+}:
+let
+  version = "1.7.0";
+  src = fetchFromGitHub {
+    owner = "vercel";
+    repo = "turbo";
+    rev = "v${version}";
+    sha256 = "YTuEv2S3jNV2o7HJML+P6OMazgwgRhUPnd/zaTWfDWs=";
+  };
+
+  go-turbo = buildGoModule rec {
```

Comment:
Why go-turbo is still kept? Isn't is replaced by the rust version? If not we should keep it as a separate package.

Reply:
The rust version has become a gateway of sorts to the go version, if you try to run the go version directly it has an error saying it should be executed from the rust version. I guess this is their transition strategy.